sos/report/plugins/openstack_mistral.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2022 Red Hat, Inc.
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin
+import os
+
+
+class OpenStackMistral(Plugin):
```

Comment:
This needs to also have at least either `IndependentPlugin` to be enabled by default across distributions, or a distro tagging class to enable on specific distributions, e.g. `RedHatPlugin`